int32 class_id
string class_name
float32 score

# Oriented Bounding Box (xywhr format)
# (x_center, y_center): 矩形框中心点坐标
# (width, height): 矩形框的宽和高
# angle: 旋转角度 (单位: 弧度)
float32 x_center
float32 y_center
float32 width
float32 height
float32 angle